docroot/themes/custom/twentyseventeen/sass/components/404_notfound/404notfound.twig

```diff
@@ -0,0 +1,11 @@
+<div class="notificationmessage__wrapper">
+ <div class="notificationtitle">
+ <img class="notification_title" src="{{ notfoundtext }}" alt="{{alt-text2}}" typeof:"foaf:Image">
```

Comment:
It's a little confusing to have `notificationtitle` and `notification_title` as classes. I think better BEM naming would be 'notification__title'. Throughout, I would use classes like `notification__message` and `notification__image`. Also, I would add a class to each element to avoid having use multiple selectors. So `notification__message-title` instead of `notification__message-wrapper h1`.